WebDriverAgentLib/Categories/XCUIApplication+FBHelpers.m

```diff
@@ -205,6 +205,9 @@ + (NSDictionary *)dictionaryForElement:(id<FBXCElementSnapshot>)snapshot
       FBExclusionAttributeFrame: ^{
```

Comment:
would it still be possible to refactor the logic related to adding of additional attributes into a separate method? 

Something like  
```
+ (NSDictionary *)addExcludableAttributesForElement:(id<FBXCElementSnapshot>)snapshot 
                             info:(NSMutableDictionary *)info
                    excludedAttributes:(nullable NSSet<NSString *> *)excludedAttributes
```?